### DISCUSSION TITLE
Connecting a search results to both a filter and another search results

### DISCUSSION THREAD

**Discussion by erosu-hab** [microsoft-search/pnp-modern-search#2238] [Q&A]
I have a search page where I set up two search results, both connected to a filter and a search box (version 4.6.1). 
I turned on the selection on one of the search results (data terms) and set up the other search results as destination (reports).
![search](https://user-images.githubusercontent.com/57365537/179656858-fc9d32bb-71c2-4582-950e-83184c08e0df.png)
The connection between the search results works, in that the items for the first page and the total number of results are returned
 correctly in the destination search results. However, when navigating on the destination web part to a page other than the first page, the page number changes but the results are incorrect. No matter which page I navigate to, I always get the results for the first page.
I inspected with developer tools the posted query and it seems that the query always starts at row 0 regardless which page I'm on; start row does not change according to page number and the number of items per page.

The issue seems to be related to the fact that the destination web part is connected to both the filter web part and the other search results and it does not matter if any value is selected in the filter on the left or not. If I disconnect the destination web part from the filter web part and leave the connection to the source web part, the paging starts to work as expected. 

I am not sure whether this is a bug, or it is because it is an invalid use case. Is the scenario described above a valid one? Ideally, we'd like to be able to apply selection and filter at the same time the results in the destination web part. 
Thank you!

**Comment by wobba** [comment]
@FranckyC is this a supported scenario?

**Reply by FranckyC** [reply]
I did not test the scenario to be honest and I suppose, from what you are describing, the code does not handle this very well. I could take a look see if the update does not require too many changes but for now, assume this scenario is not supported unfortunately :/.
